src/shared/src/NavigationDisplay.ts

```diff
@@ -44,8 +47,9 @@ export enum NdSymbolTypeFlags {
     FixInfo = 1 << 14,
     FlightPlan = 1 << 15,
     PwpDecel = 1 << 16,
-    PwpCdaFlap1White = 1 << 17,
-    PwpCdaFlap2White = 1 << 20,
+    PwpTopOfDescent = 1 << 17,
```

Comment:
Weren't you going to change the Constraint* flags, to Constraint, and Magenta, White, Amber, so they can be used for Pwps as well?

Reply:
As they are not needed right now we can do it later